examples/src/utils/LoadGltfUi.ts

```diff
@@ -0,0 +1,135 @@
+import * as THREE from 'three';
+import dat from 'dat.gui';
+import { Cognite3DViewer } from '@cognite/reveal';
+import { GLTFLoader, GLTF } from 'three/examples/jsm/loaders/GLTFLoader';
+import { TransformControls } from 'three/examples/jsm/controls/TransformControls';
+
+export class LoadGltfUi {
+  private readonly _gizmos: TransformControls[] = [];
+  private readonly _objects: THREE.Object3D[] = [];
+  private readonly _viewer: Cognite3DViewer;
+  private readonly _params = {
+    url: 'https://threejs.org/examples/models/gltf/DamagedHelmet/glTF/DamagedHelmet.gltf',
+    transformGizmosVisible: true,
+    scale: 1.0,
+    scaleStr: '1.0'
+  };
+
+  constructor(uiFolder: dat.GUI, viewer: Cognite3DViewer) {
+    this._viewer = viewer;
+    this.createGui(uiFolder);
+  }
+
+  private createGui(ui: dat.GUI): void {
+    const actions = {
+      loadGltf: () => this.loadGltf(this._params.url),
+      addHeadlight: () => this.addHeadlight(),
+    };
+    ui.add(this._params, 'transformGizmosVisible', true)
+      .name('Show gizmos?')
+      .onChange(value => {
+        this._gizmos.forEach(gizmo => {
+          gizmo.visible = value;
+          // gizmo.enabled = value;
+        });
+        this._viewer.requestRedraw();
+      });
+    ui.add(this._params, 'scaleStr').name('Scale models').onChange(scaleStr => {
+      try {
+        console.log(scaleStr);
```

Comment:
What is the reason for treating scale as a string and not a float? Because the float UI puts restrictions on how much you can scale?

In any case, I think we can remove the console logs here :+1: